README.md

```diff
@@ -129,6 +103,32 @@ running before and after benchmarking to inform you of benchmarking progress),
 which can be specified in the `PRE_EXECUTION_CMDS` and `POST_EXECUTION_CMDS`
 settings in your Krun config file.
 
+For example on a systemd Linux you may turn daemons off before execution with:
+
+```
+PRE_EXECUTION_CMDS = [
+    "sudo systemctl stop cron",
+    "sudo systemctl stop atd",
+    ...
+]
+```
+
+and turn them back on with:
+
+```
+POST_EXECUTION_CMDS = [
+    "sudo systemctl start cron",
+    "sudo systemctl start atd",
+    ...
+]
+```
+
+In general it is best practise to turn things back on explicitly, because after
+Krun runs the final benchmark it will not reboot the machine. If, for example,
+you put network interfaces down in `PRE_EXECUTION_CMDS`, you should put them
+back up in `POST_EXECUTION_CMDS` so that you can login to the machine after the
+final benchmark has been run.
```

Comment:
Shall we say something like

> If you are going to take down SSH and network, you are going to get this wrong many times, and you will lock yourself out. We recommend machines with remote management so that locked out machines can we recovered

Call me a pessimist if you like :)

Reply:
I don't think it really helps people -- if they have remote management, they do, and if they don't, they don't. Telling them in the README isn't going to change their hardware...

Reply:
But I can make it a bit clearer that this is something people need to be really careful about.

Reply:
Yeah, we just need to warn people that it's really easy to lock yourself out.

I do it on a regular basis!

Reply:
There's the stop cookie hack, but it might be an information overload?

We might point users at the warmup experiment for an "involved example"?

Reply:
Good idea.